taxcalc/functions.py

```diff
@@ -519,40 +530,27 @@ def TaxInc(c00100, c04470, c04100, _standard, e37717, c21060, c21040,
            II_brk4, II_brk5, II_brk6, _numextra, FDED, c02700):
 
     # c04500 = c00100 - max(c04470, max(c04100, _standard + e37717))
-    c04500 = c00100 - max(c21060 - c21040, _standard + e37717)
-    c04500 = c04500
-
-    if FDED == 1:
-        _othded = e04470 - c04470
-        c04100 = 0.
-        c04200 = 0.
-        _standard = 0.
-    else:
-        _othded = 0.
-
-    c04800 = max(0., c04500 - c04600 - e04805)
+    c04500 = max(0, c00100 - max(c21060 - c21040, _standard + e37717))
 
     # Check with Dan whether this is right!
     # if c04470 > _standard:
-    #     _standard = 0.
-
+    #    _standard = 0.
     # if c04470 <= _standard:
-    #     c04470 = 0.
+    #    c04470 = 0.
 
-    # why is this here, c60000 is reset many times?
-    if _standard > 0:
-        c60000 = c00100 - x04500
-    else:
-        c60000 = c04500
+    # if FDED == 1:
```

Comment:
`FDED`, Form of Deduction Code, seems to be not that helpful here if we were doing the optimization based on `c05800`.